generator/sbpg/targets/resources/sbp_messages_mod.rs

```diff
@@ -22,19 +22,19 @@ use self::(((p.identifier|mod_name)))::(((m.identifier|camel_case)));
 ((*- endfor *))
 use self::unknown::Unknown;
 
-#[cfg(feature = "sbp_serde")]
-use serde::{Serialize, Deserialize};
 use crate::serialize::SbpSerialize;
-use crate::framer::FramerError;
 
+#[enum_dispatch::enum_dispatch]
 pub trait SBPMessage: SbpSerialize {
     fn get_message_type(&self) -> u16;
     fn get_sender_id(&self) -> Option<u16>;
     fn set_sender_id(&mut self, new_id: u16);
-    fn to_frame(&self) -> std::result::Result<Vec<u8>, FramerError>;
+    fn to_frame(&self) -> std::result::Result<Vec<u8>, crate::FramerError>;
+    fn write_frame(&self, buf: &mut Vec<u8>) -> std::result::Result<(), crate::FramerError>;
 }
 
-#[cfg_attr(feature = "sbp_serde", derive(Serialize, Deserialize))]
+#[enum_dispatch::enum_dispatch(SbpSerialize, SBPMessage)]
```

Comment:
`enum_dispatch` lets you call functions defined in `SbpSerialize` and `SBPMessage` directly on `SBP` objects without having to convert to a trait object, which is more convenient and it's static dispatch not dynamic

Reply:
Changed it so we just generate the code that enum dispatch would generate. It brings down the compile time quite a bit